app/src/main/java/com/internship/move/MoveApp.kt

```diff
@@ -12,6 +14,7 @@ class MoveApp : Application() {
     override fun onCreate() {
         super.onCreate()
         initKoin()
+        appContext = applicationContext
```

Comment:
I see you're using static app context in the repository to compress the file
Since we implementet di using `koin`, let's use its power and create a `FileCompressor` class that is injected in the repo and this class has the context injected in its constructor.
This way you let koin handle the context and you keep your app implementation consistent and less error prone